backends/pipeline_api.py

```diff
@@ -5,10 +5,16 @@
 from datetime import datetime
 from urllib.parse import urljoin
 
+from rolepermissions.verifications import has_role
+
```

Comment:
The imports are broken up into three groups, the space separates the imports from third party libraries from the imports from our own code: https://www.python.org/dev/peps/pep-0008/#imports